packages/device-client/src/lib/device.store.test.ts

```diff
@@ -0,0 +1,210 @@
+import { http, HttpResponse } from 'msw';
+import { setupServer } from 'msw/node';
+import { deviceClient } from './device.store';
+
+import {
+  MOCK_PUSH_DEVICES,
+  MOCK_BINDING_DEVICES,
+  MOCK_OATH_DEVICES,
+  MOCK_DELETED_OATH_DEVICE,
+  MOCK_WEBAUTHN_DEVICES,
+} from './mock-data/msw-mock-data';
+
+// Create handlers
+export const handlers = [
```

Comment:
I will admit, we can probably abstract this handlers into a new file for reuse, but i used it twice and i'm not sure i'll have to ever use it again other than adding to it...not sure its worth it.